specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json

```diff
@@ -1359,13 +1664,23 @@
           "$ref": "#/definitions/SslEnforcement",
           "description": "Enable ssl enforcement or not when connect to server."
         },
+        "minimalTlsVersion": {
+          "$ref": "#/definitions/MinimalTlsVersion",
+          "description": "Enforce a minimal Tls version for the server."
+        },
+        "byokEnforcement": {
+          "type": "string",
+          "description": "Status showing whether the server enabled for customer managed encryption key.",
```

Comment:
I assume this shows whether the service allows the customer to manage their own key.  Status showing whter the service allows customer managed encryption keys. (or similar)

Reply:
Status showing whether the server data encryption is enabled with customer-managed keys.